README.md

```diff
@@ -69,9 +73,38 @@ The supported metadata keys for omigami are (case insensitive):
 - "inchikey_inchi"
 
 #### Notebooks
-You can find a [tutorial](https://github.com/omigami/romigami/blob/master/notebook/Tutorial.ipynb) notebook in the `/notebooks/` folder.
+You can find a [tutorial](https://github.com/omigami/romigami/blob/master/notebook/spec2vec/tutorial.ipynb) notebook in the `/notebooks/` folder.
+
+And an [R Markdown document](https://github.com/omigami/romigami/blob/master/vignettes/spec2vec/tutorial.Rmd) on the `/vignettes/` folder.
+
+### MS2DeepScore
+#### Quickstart
+
+```r
+library(romigami)
+
+omigami <- omigami_init()
+client <- omigami$MS2DeepScore(token="my_token")
+
+mgf_file_path <- "path_to_file.mgf"
+n_best_matches <- 10
+include_metadata <- list("Smiles", "Compound_name")
+ion_mode <- "positive"  # either positive or negative
+
+results <- client$match_spectra_from_path(
+                                   mgf_path = mgf_file_path,
+                                   n_best = n_best_matches,
+                                   include_metadata = include_metadata,
+                                   ion_mode = ion_mode
+)
+  ```
+
+#### Notebooks
+You can find a [tutorial](https://github.com/omigami/romigami/blob/master/notebook/ms2deepscore/tutorial.ipynb) notebook in the `/notebooks/` folder.
+
+And an [R Markdown document](https://github.com/omigami/romigami/blob/master/ms2deepscore/spec2vec/tutorial.Rmd) on the `/vignettes/` folder.
+
```

Comment:
The Notebooks section is duplicated

Reply:
Actualy one is for the MS2Deep and another one for Spec2vec. 

They are in different sessions with different links